src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -44,30 +45,41 @@ import { WrappedColumn } from './wrapped-column';
               <ng-content></ng-content>
           </ng-template>
 
-          <button class="datagrid-column-title" *ngIf="sortable" (click)="sort()" type="button">
-              <ng-container *ngTemplateOutlet="columnTitle"></ng-container>
+          <button 
+            class="datagrid-column-title" 
+            [attr.aria-label]="ariaButtonLabel"
```

Comment:
It would be nice to have a test that verifies this.

Reply:
In progress :)